### PR TITLE
Drop gmp scanner type from GSA

### PR DESCRIPTION
**Pull request by saberlynx**
**What**:

Drops GMP_SCANNER_TYPE everywhere in GSA.

**Why**:

**How**:

Tests are adjusted accordingly and should pass.

**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/web/pages/performance/performancepage.js [hunk 1]
```diff
@@ -393,10 +386,10 @@ const mapStateToProps = (rootState, {gmp}) => {
 
   if (gmp.settings.enableGreenboneSensor) {
     SLAVE_SCANNER_FILTER = Filter.fromString(
-      'type=' + GMP_SCANNER_TYPE + ' type=' + GREENBONE_SENSOR_SCANNER_TYPE,
+      'type=' + GREENBONE_SENSOR_SCANNER_TYPE,
     );
   } else {
-    SLAVE_SCANNER_FILTER = Filter.fromString('type=' + GMP_SCANNER_TYPE);
+    SLAVE_SCANNER_FILTER = Filter.fromString('');
```
Comment (sarahd93) [inline gsa/src/web/pages/performance/performancepage.js:392]: I would expect this filter to return all scanners and I think we want to return none in this case, right? I might be wrong, though.

Comment (bjoernricks) [reply gsa/src/web/pages/performance/performancepage.js:392]: The whole else clause should be removed.

File: gsa/src/web/pages/performance/performancepage.js [hunk 2]
```diff
@@ -376,10 +369,10 @@ const mapDispatchToProps = (dispatch, {gmp}) => {
...
   }
```
Comment (bjoernricks) [inline gsa/src/web/pages/performance/performancepage.js:376]: The else clause should be removed here too

File: gsa/src/web/pages/audits/row.js [hunk 3]
```diff
@@ -109,14 +109,15 @@ const Row = ({
             {entity.alterable === 1 && (
               <AlterableIcon size="small" title={_('Audit is alterable')} />
             )}
-            {isDefined(scanner) && scanner.scannerType === GMP_SCANNER_TYPE && (
```
Comment (bjoernricks) [inline gsa/src/web/pages/audits/row.js:112]: Oh I think this is wrong in 20.08 already and should include the GREENBONE_SENSOR_TYPE in that version too.

Comment (saberlynx) [reply gsa/src/web/pages/audits/row.js:112]: Yeah, we forgot to update the audit row when we added the greenbone sensor type

File: gsa/src/web/pages/performance/performancepage.js [hunk 4]
```diff
@@ -376,10 +369,8 @@ const mapDispatchToProps = (dispatch, {gmp}) => {
 
   if (gmp.settings.enableGreenboneSensor) {
```
Comment (bjoernricks) [inline gsa/src/web/pages/performance/performancepage.js:370]: I've taken a look at the code again to be sure we are doing the correct thing with the filter here. IMHO you can just remove the if here and below to always use `Filter.fromString('type=' + GREENBONE_SENSOR_SCANNER_TYPE)`
Maybe also rename the `SLAVE_SCANNER_FILTER` to `SENSOR_SCANNER_FILTER`

File: gsa/src/web/pages/performance/performancepage.js [hunk 5]
```diff
@@ -393,10 +384,8 @@ const mapStateToProps = (rootState, {gmp}) => {
...
```
Comment (bjoernricks) [inline gsa/src/web/pages/performance/performancepage.js:385]: This if should be unnecessary as mentioned above too.

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**